cves/CVE-2017-5648.yml

```diff
@@ -43,12 +43,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
-  While investigating bug 60718, it was noticed that some calls application
-  listeners did not use the appropriate facade object. when running an untrusted
-  application under a security manager, it was possible for that application to
-  retain a references to the request or response objects and thereby acces or
-  modify information associated with another web application.
+description: |
+  While investigating bug 60718, it was noticed that some calls to application
+  listeners in Apache Tomcat 9.0.0.M1 to 9.0.0.M17, 8.5.0 to 8.5.11, 8.0.0.RC1
+  to 8.0.41, and 7.0.0 to 7.0.75 did not use the appropriate facade object.
+  When running an untrusted application under a SecurityManager, it was therefore
+  possible for that untrusted application to retain a reference to the request or
+  response object and thereby access and/or modify information associated with
+  another web application. This issue came about due to requestDestroyed not
+  always being called after client disconnects.
```

Comment:
I see you mention what the fix was under the fixes section, could be worth adding a sentence here to explain how this vulnerability was resolved.